### ISSUE TITLE
btnController.reset does not display animation

### ISSUE THREAD

**Issue by Julien-Deudon** [chrisedg87/flutter_rounded_loading_button#6]
When I do a controller.reset(), the button returns to the initial position but without animation. Any solution ? Thanks.

**Comment by chrisedg87** [comment]
Hi @Julien-Deudon. Tested and reset() does trigger the desired animation. I've updated the example, please take a look and let me know.

**Comment by louia** [comment]
I think, what he means is that when tou tried this :
```  void _doSomething() async {
    Timer(Duration(seconds: 3), () {
      _btnController.success();
      Timer(Duration(seconds: 1), () {
        _btnController.reset();
      });
    });
  }
```
the _btnController.success() animation just appear at the first time but after it swipe the animation and return directly to the text of the button.

**Comment by chrisedg87** [comment]
Sorry I see what you mean. Will investigate.

**Comment by chrisedg87** [comment]
Fixed in v 1.0.6
